src/Resources/views/BlockAdmin/compose_preview.html.twig

```diff
@@ -7,63 +7,75 @@
     <a class="page-composer__container__child__edit"
        href="{{ blockAdmin.generateUrl('edit', { 'id': child.id, 'composer': true }) }}"
     >
-        {% set service = attribute(blockServices, child.type) %}
-        {% if service.metadata is defined %}
-            {% set metadata = service.metadata %}
+        {% set service = attribute(blockServices, child.type) ?? null %}
```

Comment:
well if it's the fix, I guess you can provide this for block bundle

Reply:
you talk the front end issue ? yes I can propose a PR into the block bundle

Reply:
Hey @JeromeEngelnAdeliom  after your PR in block bundle pass and mine here pass, I'll check again, But guess this issue can be handled without change the template

Reply:
@eerison I'm not sure (but not completely verified), because into the compose preview we dont pass into the front-end renderer.
And we need to keep the removed block into the composer view to allow deletion.

Reply:
Ok let's go there!

maybe the best solution should be skip child.type that doesn't exist, in this case you don't need to validade in many places if the service exist or not!

what do you think?